src/player.cpp

```diff
@@ -3626,6 +3628,45 @@ void player::apply_damage(Creature *source, body_part hurt, int dam)
     if( dam > get_painkiller() ) {
         on_hurt( source );
     }
+
+    // remove healing effects if damaged
+    int remove_med = roll_remainder( dam / 5.0f );
+    int bandaged_intensity = 0;
+    int disinfected_intensity = 0;
+
+    if( remove_med > 0 && has_effect( effect_bandaged, hurt ) ) {
+        auto matching_map_bandaged = effects->find( effect_bandaged );
+        auto &bodyparts = matching_map_bandaged->second;
+        auto found_effect = bodyparts.find( hurt );
+        effect &e_bandaged = found_effect->second;
+        bandaged_intensity = e_bandaged.get_intensity();
+        if( remove_med >= bandaged_intensity ) {
+            remove_effect( effect_bandaged, hurt );
+            remove_med -= bandaged_intensity;
+            add_msg_if_player( m_bad, ( "Bandages on your %s was destroyed!" ), body_part_name( hurt ) );
```

Comment:
Please add `_` for translation here and below.

Reply:
Done.